python/py-fqdn/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-fqdn
+version             1.5.1
+revision            0
+categories-append   devel
+license             MPL-2
+supported_archs     noarch
+
+python.versions     38 39 310 311 312
+python.pep517       yes
```

Comment:
this is the default for Python 3.7+ subports, so it will be always used here ; please remove

Reply:
I've seen different behaviour with and without `pep517 yes` present with other ports, but I'll remove it

Reply:
Yes, that is for historical reasons and no longer needed after the change in the `python` PortGroup where it is now set as default for Python 3.7+